bin/alpen-cli/src/cmd/deposit.rs

```diff
@@ -132,7 +119,7 @@ pub async fn deposit(
     // <recovery_address_pk>
     // <alpen_address>
     let magic_bytes = settings.magic_bytes.as_bytes();
-    let recovery_address_pk_bytes = recovery_address_pk.serialize();
+    let recovery_address_pk_bytes = public_key.serialize();
```

Comment:
NACK!
Before we were serializing an `XOnlyPublicKey` without the parity byte, `{02,03}`.
Now you're serializing a `PublicKey` which has the parity byte.
Just call the [`XOnlyPublicKey::from(public_key)`](https://docs.rs/bitcoin/latest/bitcoin/struct.XOnlyPublicKey.html#impl-From%3CPublicKey%3E-for-XOnlyPublicKey)